pythonforandroid/recipes/pandas/__init__.py

```diff
@@ -3,10 +3,10 @@
 
 
 class PandasRecipe(MesonRecipe):
-    version = 'v2.2.1'
-    url = 'git+https://github.com/pandas-dev/pandas'  # noqa
+    version = 'v2.3.0'
+    url = 'git+https://github.com/pandas-dev/pandas'
     depends = ['numpy', 'libbz2', 'liblzma']
-    hostpython_prerequisites = ["Cython~=3.0.5"]  # meson does not detects venv's cython
+    hostpython_prerequisites = ["Cython<4.0.0a0", "versioneer", "numpy"]  # meson does not detects venv's cython
```

Comment:
We're really that optimistic that works with Cython<4.0.0 ?

Reply:
Ok, if they're optimistic on their package, we're too on them behalf 😅

Agree on keeping 4.0.0a0